### ISSUE TITLE
Australian User - Anyone been able to get this working?

### ISSUE THREAD

**Issue by someevil** [itchannel/fordpass-ha#546]
hey, 
I have been reading through many of the issue threads here, I am not yet able to work out how to log ino the fordpass item in my HA. Anyone else from Australia region got this working yet?

**Comment by bytePatrol** [comment]
We are all waiting for the developer to push out a new update, however, try to keep in mind he does this for free in his spare time so it's not going to be an understandable priority.    I can say that before the API changes, this worked amazing in Home Assistant.

**Comment by Brian-lub** [comment]
Follow the guide on how to obtain the token, 
Login normally online, then press F12 and pass in the url.
Look for the token, it is there despite the page not logging in.

**Comment by itchannel** [comment]
Yes it does work in Australia, as per what Brian said should work. Just be aware that due to Fords constant changes I can't guarantee how long until they change something again.

**Comment by GazTheGeek** [comment]
Yes I got it working fine on 2 vehicles (Raptor and MachE) but only kept working for a day or two then stopped working - if I go through the token process again it works ok but again only for a day or 2. So not practically useful yet.

**Comment by someevil** [comment]
> Yes it does work in Australia, as per what Brian said should work. Just be aware that due to Fords constant changes I can't guarantee how long until they change something again.
